assets/airports/kewr.json

```diff
@@ -0,0 +1,5706 @@
+        "SWEET": [40.82827222, -74.92431944],
+        "TALCO": [42.06276944, -74.10773611],
+        "TOOBN": [38.99841944, -76.99050833],
+        "TOWIN": [40.53521111, -75.40037778],
+        "TUSKY": [43.565, -67],
+        "TWIGS": [42.80947222, -82.55286111],
+        "ULW"  : [42.09415556, -77.02480556],
+        "UPPRR": [41.24406944, -80.13883056],
+        "VIEEW": [42.43946389, -77.02591667],
+        "VILLS": [39.301075, -75.11052778],
+        "WAVEY": [40.23458333, -73.39437778],
+        "WEETS": [41.85749444, -74.19764167],
+        "WHITE": [40.00675556, -74.25128056],
+        "WATJA": [40.20251944, -74.49487778],
+        "WIGAN": [42.08509167, -73.81496944],
+        "WNDYL": [39.97901944, -74.81626667],
+        "WOZEE": [42.93379167, -78.73878889],
+        "YOCKY": [37.65410833, -84.15623611],
+        "YYOST": [41.58271667, -75.85584722],
+        "ZIMMZ": [40.80305556, -75.12361111]
+},
+    "restricted": [
+    ],
+    "runways":[
+        {
+            "name": ["4L", "22R"],
+            "end": [
+                [40.68165556, -74.17546111, "10ft"],
+                [40.699, -74.16443611, "9ft"]
+            ],
+            "ils": [true, true]
+        },
+        {
+            "name": ["4R", "22L"],
+            "end": [
+                [40.680525, -74.172375, "11ft"],
+                [40.69786111, -74.16135278, "11.8ft"]
+            ],
+            "ils": [true, true]
+        },
+        {
+            "name": ["11", "29"],
+            "end": [
+                [40.70280556, -74.18070833, "17ft"],
+                [40.70125278, -74.15734722, "10ft"]
+            ],
+            "ils": [true, false]
+        }
+    ],
+    "airways": {
+        "Q75" : ["BIGGY", "COPES", "MXE", "STOEN", "SACRI", "MURPH", "TOOBN", "HAMMZ", "GVE"],
+        "V3"  : ["BIGGY", "HARRS", "MAZIE", "DASDE", "MUDNE", "FROSE", "MXE"],
+        "Q436": ["COATE", "LAAYK", "YYOST", "DGRAF", "MTCAF", "REBBL", "REXXY", "HERBA", "RAAKK"],
+        "Q438": ["RAAKK", "CFKHS", "FARGN", "ICHOL", "JAAJA", "CFJHV", "TWIGS", "BERYS", "RUBYY"],
+        "Y481": ["DIXIE", "OWENZ", "POPPN", "OHRYN"],
+        "J209": ["WHITE", "CYN", "VILLS", "SBY"],
+        "J79" : ["SBY", "LEESA"],
+        "Q818": ["GAYEL", "MSLIN", "STOMP", "BUFFY", "CFB", "VIEEW", "KELIE", "WOZEE"],
+        "J48" : ["LANNA", "PTW", "BYRDD", "HAAGN", "PENSY", "EMI", "CSN", "MOL", "FLASK"],
+        "J223": ["NEION", "HOPCE", "CORDS"],
+        "J60" : ["NEWEL", "GYNTS", "DANNR", "FORTT", "DOOTH", "PSB"],
+        "J6"  : ["PARKE", "TOWIN", "KAYNE", "HOUTN", "LRP", "SAAME", "FLIRT", "HYPER", "MRB", "EYTEE", "COLNS", "PERKS", "HVQ"],
+        "Q68" : ["HVQ", "SPAYD", "YOCKY"],
+        "Q42" : ["ZIMMZ", "SPOTZ", "MIKYG", "HOTEE", "BRNAN", "PSYKO", "BUBBA", "HIDON"]
+    },
+    "sids": {
+        "EWR4": {
+            "icao": "EWR4",
+            "name": "Newark Four",
+            "rwy": {
+                "KEWR4L" : [["_RWY04L01DME", "A05+"], "_RWY04L04DME", ["#290", "A030"]],
```

Comment:
![image](https://user-images.githubusercontent.com/45713872/147397923-a05457f3-7ccc-4097-81be-ec049eededce.png)

Reply:
The top altitudes are different for each runway, and generally assigned at clearance time. The text description in the chart mentions the altitudes used in the file. The aircraft appear to handle the altitude restriction on the heading segment just fine, but I am open to handling it differently if you have suggestions.

Reply:
Ahh, I see. If it ain't broke, don't fix it, I guess. I can't think of a better way of writing it so it can stay, I just don't like it on principle 😅